schema.sql

```diff
@@ -21,3 +21,27 @@ CREATE TABLE music_albums (
   FOREIGN KEY (source_id) REFERENCES sources(id),
   FOREIGN KEY (label_id) REFERENCES labels(id)
 );
+
+-- Create the 'label' table
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(255),
+  color VARCHAR(255),
+);
+
+-- Create the 'book' table
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  publish_date DATE,
+  archived BOOLEAN,
+  publisher VARCHAR(255),
+  cover_state VARCHAR(255),
+  genre_id INT NOT NULL,
+  author_id INT NOT NULL,
+  source_id INT NOT NULL,
```

Comment:
I think we need to remove the `source_id` reference, because we just have 'genre', 'author' and 'label' classes.